deployments/rialto/docker-compose.yml

```diff
@@ -93,13 +93,13 @@ services:
       - "9618:9616"
     depends_on: *all-nodes
 
-  bridge-node-alice: &bridge-node
-    image: paritytech/bridge-node
+  rialto-bridge-node-alice: &rialto-bridge-node
+    image: paritytech/rialto-bridge-node
```

Comment:
I just tried the deployment locally and this change is actually a problem right now. We don't have any published images called `paritytech/rialto-bridge-node`, they're still published as `paritytech/bridge-node`. This means that when I do `docker-compose pull` this fails. Let's change this back to `bridge-node`, and maybe next week when General-Beck comes back we'll get some new images published.

Reply:
I don't think anything should be done here - images are auto-published from master every day, right? So once this PR is merged, it'll be auto-fixed next day.

Reply:
Yeah, good point. It escaped me that published package names would be whatever is in the `publish-docker.yml` file